src/snowflake/cli/_plugins/cortex/commands.py

```diff
@@ -48,6 +51,23 @@
 
 SEARCH_COMMAND_ENABLED = sys.version_info < PYTHON_3_12
 
+SOURCE_EXCLUSIVE_OPTION_NAMES = ["text", "file", "source_document_text"]
+
+ReadableFileOption = OverrideableOption(
```

Comment:
Need documentation for this option, and TextSourceArgument. When to use them, and what they do.